backend/functions/lib/adapters/gateways/dxtr/orders.dxtr.gateway.ts

```diff
@@ -66,7 +67,11 @@ export class DxtrOrdersGateway implements OrdersGateway {
 
       orderSync.orders = rawOrders
         .map((rawOrder) => {
-          if (orderSync.maxTxId < rawOrder.txId) orderSync.maxTxId = rawOrder.txId;
+          const txIdAsInt = parseInt(rawOrder.txId);
+          if (maxTxIdasInt < txIdAsInt) {
+            orderSync.maxTxId = rawOrder.txId;
+            maxTxIdasInt = txIdAsInt;
+          }
```

Comment:
Maybe simplify this. ``` if (maxTxIdAsInt < txIDAsInt) maxTxIdasInt = txIDasInt;```
Then just assign the maxTxIdasInt to the orderSync object once before returning.

Reply:
See Fritz's note on variable naming above.